src/PowerJoinClause.php

```diff
@@ -175,15 +161,16 @@ public function whereNull($columns, $boolean = 'and', $not = false)
 
     public function newQuery(): self
     {
-        return new static($this->newParentQuery(), $this->type, $this->table, $this->model); // <-- The model param is needed
+        return new static($this->newParentQuery(), $this->type, $this->table,
+            $this->model); // <-- The model param is needed
```

Comment:
Is this some kind of code style tool? Otherwise it may be better to break this into one line per parameter. I'm planning to implement php-cs-fixer here

Reply:
You are right. I reverted this kind of changes